spec/node/asciidoctor.spec.js

```diff
@@ -9,11 +9,20 @@ var config = {
   }
 };
 var asciidoctor = require('../../build/asciidoctor.js')(config);
+function asciidoctorVersionGreaterThan (version) {
```

Comment:
Let's follow up on this in a separate issue about getAsciidoctorVersion and perhaps a comparison helper method for comparing gem versions.